crates/txtx-core/src/eval/mod.rs

```diff
@@ -1068,7 +1297,7 @@ pub enum CommandInputEvaluationStatus {
 }
 
 pub fn perform_inputs_evaluation(
-    command_instance: &CommandInstance,
+    with_evaluatable_inputs: &impl WithEvaluatableInputs,
```

Comment:
For this function I'm mostly just changing things to use the `WithEvaluatableInputs` impl rather than a command instance